test/CMakeLists.txt

```diff
@@ -108,7 +116,7 @@ add_executable(test_xeus ${XEUS_TESTS} ${XEUS_HEADERS})
 if(DOWNLOAD_GTEST OR GTEST_SRC_DIR)
     add_dependencies(test_xeus gtest_main)
 endif()
-target_link_libraries(test_xeus xeus ${GTEST_BOTH_LIBRARIES} Threads::Threads ${nlohmann_json_TARGET})
+target_link_libraries(test_xeus ${xeus_TARGET} ${GTEST_BOTH_LIBRARIES} Threads::Threads ${nlohmann_json_TARGET})
```

Comment:
I don't think you can have a xeus_TARGET macro since it not one or the other.

Reply:
Hi! I set `xeus_TARGET` above(L98-L104). I type these words on my Android and the input method is  driving me crazy.